.travis.yml

```diff
@@ -30,7 +30,7 @@ before_install:
   - export PATH="$PWD/_meson:$PATH"
 
 install:
-  - meson.py -DMUTE_GEOLOG=true -DMUTE_GEOTIMER=true build
+  - meson.py -DMUTE_GEOLOG=true -DMUTE_GEOTIMER=true -DMATH_OPTIM=true build
```

Comment:
remove `-DMATH_OPTIM=true` and we are done

Reply:
done